node/impl/full/eth.go

```diff
@@ -1912,6 +1907,7 @@ func newEthTxReceipt(ctx context.Context, tx ethtypes.EthTx, lookup *api.MsgLook
 					continue
 				}
 				if entry.Key == ethtypes.EthTopic1 || entry.Key == ethtypes.EthTopic2 || entry.Key == ethtypes.EthTopic3 || entry.Key == ethtypes.EthTopic4 {
```

Comment:
I assume that Eth blooms don't factor in the position of the topic? If so, this is fine.

Reply:
Correct, position doesn't matter